dpsim/src/DenseLUAdapter.cpp

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2017-2021 Institute for Automation of Complex Power Systems,
+ *                     EONERC, RWTH Aachen University
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *********************************************************************************/
+
+#include <dpsim/DenseLUAdapter.h>
+
+using namespace DPsim;
+
+namespace DPsim
+{
+
+    DenseLUAdapter::~DenseLUAdapter() = default;
+
+    void DenseLUAdapter::preprocessing(SparseMatrix& systemMatrix, std::vector<std::pair<UInt, UInt>>& listVariableSystemMatrixEntries)
+    {
+        /* No preprocessing phase needed by PartialPivLU */
+    }
+
+    void DenseLUAdapter::factorize(SparseMatrix& systemMatrix)
+    {
+		/* only a simple dense factorization */
```

Comment:
Here no comment required, as this is expected

Reply:
done